source/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp

```diff
@@ -66,12 +61,32 @@ int main()
         std::cout << "RotationMatrix:\n" << transformationMatrix.linear() << std::endl;
         std::cout << "TranslationVector:\n" << transformationMatrix.translation() << std::endl;
 
-        // Convert from Zivid to Robot (Transformation Matrix --> any format)
-        const auto robotRotationRepresentations = zividToRobot(transformationMatrix);
-
-        // Convert from Robot to Zivid (any format --> Rotation Matrix)
-        const auto transformationMatrix2 =
-            robotToZivid(robotRotationRepresentations, transformationMatrix.translation());
+        /*
+         * Convert from Zivid to Robot (Transformation Matrix --> any format)
+         */
+        print_header("Convert from Zivid (Rotation Matrix) to Robot");
+        const Eigen::AngleAxisd axisAngle(transformationMatrix.linear());
+        std::cout << "AxisAngle:\n" << axisAngle.axis() << ", " << axisAngle.angle() << std::endl;
+        const auto rotation_vector = rotation_matrix_to_rotation_vector(transformationMatrix.linear());
+        std::cout << "Rotation Vector:\n" << rotation_vector << std::endl;
+        const Eigen::Quaterniond quaternion(transformationMatrix.linear());
+        std::cout << "Quaternion:\n" << quaternion.coeffs() << std::endl;
+        const auto rpy_list = rotation_matrix_to_list_of_roll_pitch_yaw(transformationMatrix.linear());
+        
+        /*
+         * Convert from Robot to Zivid (any format --> Rotation Matrix (part of Transformation Matrix))
+         */
+        print_header("Convert from Robot to Zivid (Rotation Matrix)");
+        const auto rotation_matrix_from_axis_angle = axisAngle.toRotationMatrix();
+        std::cout << "Rotation Matrix from Axis Angle:\n" << rotation_matrix_from_axis_angle << std::endl; 
+        const auto rotation_matrix_from_rotation_vector = rotation_vector_to_rotation_matrix(rotation_vector);
+        std::cout << "Rotation Matrix from Rotation Vector:\n" << rotation_matrix_from_rotation_vector << std::endl;
+        const auto rotation_matrix_from_quaternion = quaternion.toRotationMatrix();
+        std::cout << "Rotation Matrix from Quaternion:\n" << rotation_matrix_from_quaternion << std::endl;
+        list_of_roll_pitch_yaw_to_rotation_matrix(rpy_list);
+
+        Eigen::Affine3d transformationMatrix2(rotation_matrix_from_quaternion);
```

Comment:
Can you find a more explanatory name for this variable? There's a difference between `transformationMatrix` and `transformationMatrix2` which is probably more important than being number one and number two?